src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -399,7 +399,7 @@ function validateConstSumQuestions(){
             var qnNum = constSumQuestionNums[i];
             
             // indicate the question number where the errors are located at
-            if($("p[id^='constSumMessage-"+qnNum+"-'].text-color-red").length > 0){
+            if($("p[id='constSumMessage-"+qnNum+"-'].text-color-red").length > 0){
```

Comment:
This looks wrong. The selector will now fail to match the correct element, since the response index is after the '-'.